src/DynamicExpresso.Core/Parsing/Parser.cs

```diff
@@ -27,6 +27,7 @@ public static Expression Parse(ParserArguments arguments)
 		private const NumberStyles ParseLiteralNumberStyle = NumberStyles.AllowLeadingSign;
 		private const NumberStyles ParseLiteralUnsignedNumberStyle = NumberStyles.AllowLeadingSign;
 		private const NumberStyles ParseLiteralDecimalNumberStyle = NumberStyles.AllowLeadingSign | NumberStyles.AllowDecimalPoint;
+        private const NumberStyles ParseLiteralDoubleNumberStyle = NumberStyles.AllowLeadingSign | NumberStyles.AllowDecimalPoint | NumberStyles.AllowExponent;
```

Comment:
Can you just align the code using the same indentation?

Reply:
code aligned

Reply:
>I have the feealing that in some cases we can have problems ... (like having a math expression with + or - ...)

in my project  i use expression like that
@Var1 + @Var2 / Math.Pow(2, 2)
where @Var1 = 2e+10 and @Var2 = 0.0006
with fix which i've made, all my expressions calculated fine

>do you think that it makes sense as a default or we should put a configuration instead?

i don't think so, because exponential format is one default formats of number reprecentation, you can write 0.0003 or 3e-4  and this are same numbers

Reply:
>can you add a some more tests where we use this notation inside more complex expressions?

where i can add new tests, which class i should extends. I can add new test to GithubIssues.cs

Reply:
Very good! Thank you! In the week-end I will do a final check and release a new version.